.gitignore

```diff
@@ -8,4 +8,6 @@ target
 
 # we don't want to push this file,
 # each contributor must create its own to signal acceptance
-akka-projection-jdbc/src/test/resources/container-license-acceptance.txt
\ No newline at end of file
+akka-projection-jdbc/src/test/resources/container-license-acceptance.txt
+akka-projection-slick/src/test/resources/container-license-acceptance.txt
```

Comment:
Isn't there a shotrcut as in `*/src/test/resources/container-license-acceptance.txt`?
Or did you want to explicitly list the allowed copies?

Reply:
(not a blocker)

Reply:
Explicit listing. I don't want to copy that in all src/test/resources folders. No harm, but better to be explicit.